src/plugins/console/server/lib/spec_definitions/json/generated/cluster.state.json

```diff
@@ -28,7 +28,7 @@
       "metrics": [
         "_all",
         "blocks",
-        "master_node",
+        "cluster_manager_node",
```

Comment:
I think cluster state metrics still have master_node, just deprecated. It will be removed at 3.0. Do we want to keep both to match OpenSearch?

Reply:
Good call! Updated.